recipe/meta.yaml

```
@@ -1,31 +1,31 @@
 {% set name = "fastapi" %}
-{% set version = "0.103.0" %}
+{% set version = "0.112.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4166732f5ddf61c33e9fa4664f73780872511e0598d4d5434b1816dc1e6d9421
+  sha256: 3d4729c038414d5193840706907a41839d839523da6ed0c2811f1168cac1798c
 
 build:
   number: 0
-  # httpx and its deps (httpcore, idna) aren't available on s390x. 
+  # httpx and its deps (httpcore, idna) aren't available on s390x.
   # httpx is required only for testing fastapi  by using `from fastapi.testclient import TestClient`
-  skip: true  # [py<37 or s390x]
+  skip: true  # [py<38 or s390x]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
  entry_points:
    - fastapi = fastapi.cli:main
```
Missing `entry_points`, see https://github.com/fastapi/fastapi/blob/0.112.2/pyproject.toml#L104. It was added in 0.112.0, see https://github.com/fastapi/fastapi/commit/a25c92ceb9ab5be49901443d7184c149797192df. That means that FastAPI has its own command instead of, for example, `uvicorn app.main:app` to run a local server

---

Ah, it will ask to install `fastapi-cli` https://github.com/fastapi/fastapi/blob/master/fastapi/cli.py#L9-L12.